src/ts/components/core/Burger.tsx

```diff
@@ -17,6 +17,8 @@ interface Props
     size?: MantineSize | (string & {}) | number;
     /** Key of `theme.colors` of any valid CSS value, by default `theme.white` in dark color scheme and `theme.black` in light */
     color?: MantineColor;
+    /** Height of the burger lines */
+    lineSize?: MantineSize | (string & {}) | number;
```

Comment:
```suggestion
    lineSize?: string | number;
```
try it like this so it matches the Mantine prop typ

![image](https://github.com/user-attachments/assets/8d0d4677-7478-421e-bf78-8f8bd885d145)

Reply:
Got it!